tests/cli/test_datasets.py

```diff
@@ -27,8 +27,100 @@
 
 from renku import cli
 from renku._compat import Path
+from renku.api.config import RENKU_HOME
+from renku.api.datasets import DatasetsApiMixin
 from renku.cli._format.dataset_files import FORMATS as DATASET_FILES_FORMATS
 from renku.cli._format.datasets import FORMATS as DATASETS_FORMATS
+from renku.models.refs import LinkReference
+
+
+def test_datasets_create_clean(data_repository, runner, project, client):
+    """Test creating a dataset in clean repository."""
+    # create a dataset
+    result = runner.invoke(cli.cli, ['dataset', 'create', 'dataset'])
+    assert 0 == result.exit_code
+    assert 'OK' in result.output
+
+    with client.with_dataset('dataset') as dataset:
```

Comment:
would be cleaner here to assert that the metadata file was created and committed.